### ISSUE TITLE
Do you plan to release "weights"?

### ISSUE THREAD

**Issue by dj-nuo** [baijinglin/TS-BSmamba2#2]
Hi! Amazing performance according to the comparison tables :-)
Do you have any plans on releasing weights? So that people can reproduce & fine-tune the results you achieved on MUSDB18

**Comment by lucellent** [comment]
On the same topic, is the repo finished or will you add more instructions, including requirements, training guide etc?
Also sadly looks like it's linux only, due to mamba... hope I'm wrong

**Comment by baijinglin** [comment]
We currently don’t have plans to release the model weights. We’re sorry if this comes as a disappointment.

> Hi! Amazing performance according to the comparison tables :-) Do you have any plans on releasing weights? So that people can reproduce & fine-tune the results you achieved on MUSDB18

We currently don’t have plans to release the model weights. We’re sorry if this comes as a disappointment.

**Comment by baijinglin** [comment]
> On the same topic, is the repo finished or will you add more instructions, including requirements, training guide etc? Also sadly looks like it's linux only, due to mamba... hope I'm wrong

sorry, I've been busy with other tasks lately. I will continue to update the repository when I have time. At the moment, there's no plan to release the training guide. If you're planning to train the mamba-2 model on multiple GPUs, I recommend using DDP (Distributed Data Parallel) instead of DP (Data Parallel). Also, it's advisable to use PyTorch version 2.0 or above.

**Comment by baijinglin** [comment]
> On the same topic, is the repo finished or will you add more instructions, including requirements, training guide etc? Also sadly looks like it's linux only, due to mamba... hope I'm wrong

Yes, it is correct that the system currently only supports Linux.
